tools/run_tests/run_tests.py

```diff
@@ -509,6 +526,7 @@ def __str__(self):
 _WINDOWS_CONFIG = {
     'dbg': 'Debug',
     'opt': 'Release',
+    'gcov': 'Release',
```

Comment:
Yes, I think you're right (but for C# it's actually not a big difference and for windows we don't have a way to collect csharp_ext coverage anyway).

As I have 2 pull requests based on this one, would it be fine to address this in a followup PR?